state_of_css.yml

```diff
@@ -537,8 +759,24 @@ translations:
     - key: opinions.sum_up_one_word_css.description
       t: ¿Cómo resumirías tu opinión de CSS en una palabra?
 
+    - key: happiness.pre_post_processors
+      t: ¿Como estás de satisfecho con el estado de los pre/post procesadores?
+    - key: happiness.css_in_js
+      t: ¿Cómo de satisfecho estás con el estado de las librerías de CSS-in-JS?
+    - key: happiness.css_frameworks
+      t: ¿Cómo de satisfecho estás con el estado de los frameworks de CSS?
+
     - key: happiness.state_of_the_web
-      t: ¿Cómo de contento estás con el estado general de las tecnologías web?
+      t: ¿Cómo de satisfecho estás con el estado general de las tecnologías web?
 
     - key: happiness.state_of_css
-      t: ¿Cómo de contento estás con el estado general de CSS?
+      t: ¿Cómo de satisfecho estás con el estado general de CSS?
+
+    ###########################################################################
+    # Resources
+    ###########################################################################
+    
+    - key: resources.first_steps
+      t: Primeros Pasos con CSS
+    - key: resources.first_steps.description
+      t: ¿Cómo has empezado a aprender CSS?
```

Comment:
Yo pondría: `¿Cómo empezaste a aprender CSS?` aunque veo que `¿Cómo has empezado a aprender...` es utilizado en otras partes de la traducción. De pronto es una diferencia entre el Español de España y el Español al que estoy acostumbrada.

Reply:
Creo que el Español que usáis vosotros es más dado al pasado simple que al pasado perfecto. Es como en Inglés. Se usa más el `went` que el `have gone`. Pero a lo mejor es sólo una manía mía. Voy a dejar la tuya para que suene menos rígida 🙂

Reply:
The simpler the better but maybe it's my English taking over my Spanish 😆